omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+data with no other ifs and buts.
+
+Four suggested workflows
+------------------------
+
+=============================== ======================= ======================= ===================== ================================
+Required Privileges             :term:`Data viewer`     :term:`Importer`        :term:`Analyst`       :term:`Group and Data Organizer`
+------------------------------- ----------------------- ----------------------- --------------------- --------------------------------
+:term:`Sudo`                                      N                     Y                      N                       N
+:term:`Write Data`                                N                     N                      Y                       Y
+:term:`Delete Data`                               N                     N                      N                       Y
+:term:`Chgrp`                                     N                     N                      N                       Y
+:term:`Chown`                                     N                     N                      Y (O)                   Y
+:term:`Create and Edit Groups`                    N                     N                      N                       Y
+:term:`Create and Edit Users`                     N                     N                      N                       Y
+:term:`Add Users to Groups`                       N                     N                      N                       Y
+:term:`Upload Scripts`                            N                     N                      Y                       N
+
+=============================== ======================= ======================= ===================== ================================
+
+Y == privilege required, checkbox in OMERO.web interface is checked
+N == privilege not required, checkbox is not checked
+(O) == this privilege is suggested as optional for the workflow
+
+.. note::
+    **Restricted admins workflows in OMERO.clients:** Please do not
+    expect for any workflows mentioned here that all OMERO.clients
+    OMERO.web, OMERO.insight, command line interface (CLI)) are fully
+    equipped to execute them (see details below). New features will
+    be added in OMERO.clients in the 5.3.x series of OMERO releases.
+
+.. note::
+    **Group allegiances** All the workflows here do not assume that
+    the administrator with restricted privileges is not a member of
+    any group except the System group. This does not preclude such
+    administrator from being a member of any number of groups.
+    Inside the groups the restricted admin is a member of, he/she
+    has the same privileges as other group members of that group
+    additionally to his/her adminstrative privileges.
+
+.. note::
+    **Deleting privileges**: :term:`Sudo` privilege has implicitly
+    in itself a Delete power over the data the administrator is
+    sudoed in as. If you want to prevent restricted admin from
```

Comment:
`sudoed in as` reads awkwardly.  data owned by the user the administrator is working on behalf of?